lib/API.jsx

```diff
@@ -327,6 +327,20 @@ export default function API(network, args) {
             return performPOSTRequest(commandName, parameters);
         },
 
+        /**
+         * Initiate a screen share request with a user.
+         *
+         * @param {object} parameters
+         * @param {Number} parameters.accountID
+         * @param {string} parameters.CallSid
```

Comment:
Makes sense to me! I think it could be useful to add a comment explaining what the `CallSid` is, I don't think it's very self-explanatory, unless it's just a "call ID" (with an extra "s" thrown in) haha